packages/node/src/local/client.ts

```diff
@@ -85,6 +113,7 @@ export class LocalEvaluationClient {
       this.flags,
     );
     const results: Results = evaluation.evaluate(this.flags, user);
+    void this.assignmentService?.track(new Assignment(user, results));
```

Comment:
filter results before passing to track.

Reply:
Oh, last thing, we always want to include flags with type `mutual-exclusion-group` or `holdout-group` in the assignment. 

This is a bit of a hack until we add topological sorting before evaluation in this SDK also :)